src/components/Menu.tsx

```diff
@@ -73,6 +73,11 @@ class Menu extends React.Component<Properties> {
         const floors = event.getFloors();
         const volunteer = event.getCurrentVolunteer();
 
+        // Determine name to for the Volunteers menu item
+        // Only show as Volunteers if you have access to more than one group.
+        const volunteerGroupName = Array.from(event.getVolunteerGroups()).length > 1 ?
+            'Volunteers' : Array.from(event.getVolunteerGroups())[0].label;
```

Comment:
nit: Sla Array.from(..) op in een temporary, dan hoeven we niet twee keer over de iterator te lopen. Dat is namelijk de common case. (Iedereen zonder admin toegang.)